### ISSUE TITLE
bin/admin/change-versions is busted

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8817]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8817|
|Type|release blocker: release process bug|
|Created|2016-09-14 20:09:27Z|
|Branch|https://github.com/twisted/twisted/tree/8817-incremental|

The change-versions script can't cope with the src/ move, but the tests pass. Since it's broken and needs a lot of ripping apart to actually work with the src/ move, it may be a good opportunity to kill two birds with one stone and move to Incremental (which is the old t.p.versions) and it's update script, to unify across the different Twisted related projects.

<details><summary>Searchable metadata</summary>

```
trac-id__8817 8817
type__release_blocker__release_process_bug release blocker: release process bug
reporter__hawkowl hawkowl
priority__high high
milestone__Twisted_16_5 Twisted 16.5
branch__8817_incremental 8817-incremental
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__None None
time__1473883767692328 1473883767692328
changetime__1476081024986375 1476081024986375
version__None None
owner__Mark_Williams__markrwilliams_____ Mark Williams <markrwilliams@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)|@hawkowl set owner to @hawkowl|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

PR: https://github.com/twisted/twisted/pull/545

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)|@hawkowl removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_5"></a>|@glyph commented|
|-|-|

"busted" how?

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_6"></a>|@glyph set owner to @hawkowl|
|-|-|

[See github review.](https://github.com/twisted/twisted/pull/545#pullrequestreview-1098825).  OMG I can actually link to github reviews now, they are discrete.  Amaze.

One more question: is this really a release blocker?  What's the bug?

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_8"></a>|@hawkowl removed owner|
|-|-|

I think I have addressed the review points, putting back up for review.

**Comment by twisted-trac** [comment]
|[<img alt="markrwilliams's avatar" src="https://avatars.githubusercontent.com/u/704247?s=50" width="50" height="50">](https://github.com/markrwilliams)<a name="note_9"></a>|@markrwilliams commented|
|-|-|

A review just for you:
https://github.com/twisted/twisted/pull/545#pullrequestreview-3444886

**Comment by twisted-trac** [comment]
|[<img alt="markrwilliams's avatar" src="https://avatars.githubusercontent.com/u/704247?s=50" width="50" height="50">](https://github.com/markrwilliams)<a name="note_10"></a>|@markrwilliams set owner to @markrwilliams<br>@markrwilliams set status to `closed`|
|-|-|

In changeset 5c1777777f85aaf799c2269f9f5dddcf0d59803e
```
#!CommitTicketReference repository="" revision="5c1777777f85aaf799c2269f9f5dddcf0d59803e"
Merge pull request #545 from twisted/8817-incremental

Author: hawkowl
Reviewer: glyph, markrwilliams
Fixes: #8817

Replace t.p.versions with incremental
```
